src/perfcollect/perfcollect

```diff
@@ -1280,10 +1292,12 @@ CreateLTTngSession()
 
 SetupLTTngSession()
 {
+    
     # Setup per-event context information.
     RunSilent "lttng add-context --userspace --type vpid"
     RunSilent "lttng add-context --userspace --type vtid"
     RunSilent "lttng add-context --userspace --type procname"
+    RunSilent "lttng add-context --kernel -t pid -t procname"
```

Comment:
Does this taking a pid imply we've started the target process before we start tracing?

Reply:
No, this is telling LTTng to add the pid and process name to every event.  "lttng add-context" allows you to add additional context information to each event that comes from a particular channel (user or kernel).